### PR TITLE
Beta-release-fixes(2)

### PR DESCRIPTION
**Pull request by inetaa**
There was Error inflating class TextView caused by vectorDrawable. "As of Android Support Library 23.3.0, support vector drawables can only be loaded via app:srcCompat". Source: https://android-developers.blogspot.lt/2016/02/android-support-library-232.html

https://trello.com/c/RbdcjwWy/103-app-craches-on-api-19-on-problem-list

### CODE REVIEW

File: app/src/main/res/layout/activity_new_problem.xml [hunk 1]
```diff
@@ -15,14 +15,31 @@
...
+                    android:layout_marginRight="@dimen/activity_horizontal_margin"
+                    android:layout_marginBottom="8dp"
+                    android:src="@drawable/ic_photo_camera_white" />
+            </FrameLayout>
 
             <GridLayout
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:layout_margin="28dp"
+                android:layout_marginRight="28dp"
```
Comment (GediminasZukas) [inline app/src/main/res/layout/activity_new_problem.xml]: Margin values doesn't follow baseline grid rules: https://material.google.com/layout/metrics-keylines.html#
Could it be`32dp` or `24dp`?

File: app/src/main/java/lt/vilnius/tvarkau/AboutActivity.java [hunk 2]
```diff
@@ -126,4 +127,27 @@ private void openEmail(String emailAddress) {
...
+        String model = Build.MODEL;
+        String androidVersion = Build.VERSION.RELEASE;
+        if (model.startsWith(manufacturer)) {
+            return " (" + capitalize(model) + ", " + "Android " + androidVersion + ")";
+        } else {
+            return " (" + capitalize(manufacturer) + " " + model + ", " + "Android " + androidVersion + ")";
+        }
+    }
+
+    private String capitalize(String s) {
```
Comment (GediminasZukas) [inline app/src/main/java/lt/vilnius/tvarkau/AboutActivity.java]: In my opinion, `getDeviceInfo()` `capitalize()` methods are util methods and semantically has nothing to do with Activity. Thus, I would suggest to move these to one of the utility class.

File: app/src/main/java/lt/vilnius/tvarkau/utils/GlobalConsts.java [hunk 3]
```diff
@@ -19,6 +19,7 @@
     public final static String CODE_FOR_VILNIUS_EMAIL = "tvarkau.vilniu@codeforvilnius.lt";
     public final static String VILNIUS_MUNICIPALITY_EMAIL = "e.vicemeras@vilnius.lt";
     public final static String VILNIUS_MUNICIPALITY_PHONE = "1664";
+    public final static String GOOGLE_PLAY_SERVICES = "https://play.google.com/store/apps/details?id=com.google.android.gms";
```
Comment (GediminasZukas) [inline app/src/main/java/lt/vilnius/tvarkau/utils/GlobalConsts.java]: I think you can use this field directly in `NewProblemActivity` if you are not going to use it any other place.

File: app/src/main/java/lt/vilnius/tvarkau/NewProblemActivity.java [hunk 4]
```diff
@@ -243,6 +246,7 @@ public void sendProblem() {
             Action1<Throwable> onError = throwable -> {
                 throwable.printStackTrace();
                 FirebaseCrash.report(throwable);
+                Crashlytics.logException(throwable);
```
Comment (vycius) [inline app/src/main/java/lt/vilnius/tvarkau/NewProblemActivity.java]: Consider abstracting logging to separate class `AppLog`. In this way we won't need to do call `FirebaseCrash::report` and `Crashlytics::logException` separately.

File: app/build.gradle [hunk 5]
```diff
@@ -57,6 +60,7 @@ repositories {
     maven { url "https://clojars.org/repo/" }
     maven { url "http://repo.commonsware.com.s3.amazonaws.com" }
     maven { url "https://jitpack.io" }
+    maven { url 'https://maven.fabric.io/public' }
```
Comment (vycius) [inline app/build.gradle:62]: Keep `https://jitpack.io` maven dependecy as the last one

File: app/fabric.properties [hunk 6]
```diff
@@ -0,0 +1,3 @@
+#Contains API Secret used to validate your application. Commit to internal source control; avoid making secret public.
```
Comment (vycius) [inline app/fabric.properties:1]: Is it ok to have this file in public Github repo ?

File: app/src/main/java/lt/vilnius/tvarkau/utils/DeviceUtils.java [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+        String model = Build.MODEL;
+        String androidVersion = Build.VERSION.RELEASE;
+        if (model.startsWith(manufacturer)) {
+            return " (" + capitalize(model) + ", " + "Android " + androidVersion + ")";
+        } else {
+            return " (" + capitalize(manufacturer) + " " + model + ", " + "Android " + androidVersion + ")";
+        }
+    }
+
+    public static String capitalize(String s) {
```
Comment (vycius) [inline app/src/main/java/lt/vilnius/tvarkau/utils/DeviceUtils.java]: Make it `private` or move to another class such as `StringUtils`

File: app/fabric.properties (hunk 6 above)
Comment (inetaa) [inline app/fabric.properties:1]: Good point! Actually we should move all the keys away from project as recommended here: https://docs.fabric.io/android/fabric/settings/working-in-teams.html. As for now I will keep this task for the future (https://trello.com/c/dYeIJQ1i/113-remove-sensitive-data-from-github).

**Pull request merged**